includes/class-zoninator-api-controller.php

```diff
@@ -101,6 +109,46 @@ function setup() {
 	}
 
 	/**
+	 * Get the list of all zones
+	 *
+	 * @param WP_REST_Request $request Full data about the request.
+	 * @return WP_Error|WP_REST_Response
+	 */
+	function get_zones( $request ) {
+		$results = $this->instance->get_zones();
+
+		if ( is_wp_error( $results ) ) {
+			return $this->_bad_request( array(
+				'message' => $results->get_error_message()
+			) );
+		}
+
+		return new WP_REST_Response( $results, 200 );
+	}
+
+	/**
+	 * @param WP_REST_Request $request Full data about the request.
+	 * @return WP_Error|WP_REST_Response
+	 */
+	function create_zone( $request ) {
+		$name = $this->_get_param( $request, 'name', '' );
+		$slug = $this->_get_param( $request, 'slug', '' );
+		$description = $this->_get_param( $request, 'description', '', 'strip_tags' );
+
+		$result = $this->instance->insert_zone( $slug, $name, array( 'description' => $description ) );
```

Comment:
Cool! Better if we split this into multiple lines (standard php practice on arrays with string keys, leave a comma in the array's end. Like so:

```php
$result = $this->instance->insert_zone( $slug, $name, array( 
    'description' => $description, 
) );
```